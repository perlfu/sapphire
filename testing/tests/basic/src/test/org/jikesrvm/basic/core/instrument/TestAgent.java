/*
 * This file is part of Jikes RVM (http://jikesrvm.sourceforge.net).
 * The Jikes RVM project is distributed under the Common Public License (CPL).
 * A copy of the license is included in the distribution, and is also
 * available at http://www.opensource.org/licenses/cpl1.0.php
 *
 * (C) Copyright Elias Naur 2006
 *
 * $Id: /jikesrvm/local/testing/tests/javaagent/src/TestAgent.java 10522 2006-11-14T22:42:56.816831Z dgrove-oss  $
 */
package test.org.jikesrvm.basic.core.instrument;

/**
 * @author Elias Naur
 */
public class TestAgent {
	public static void main(String[] args) {
		System.out.println("TestAgent executed.");
	}
}
