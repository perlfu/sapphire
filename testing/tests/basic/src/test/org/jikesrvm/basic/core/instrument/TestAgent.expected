Running premain with args: Option1
Adding class transformer
Array size ok: true
Object size ok: true
AgentX in initiated classes? true
AgentX in all classes? true
Transforming class: test/org/jikesrvm/basic/core/instrument/TestAgent
TestAgent executed.
